packages/block-library/src/button/block.json

```diff
@@ -22,14 +26,14 @@
 		"title": {
 			"type": "string",
 			"source": "attribute",
-			"selector": "a",
+			"selector": "a,button",
```

Comment:
This makes the experiment less clean as it introduces a change to the button block that is outside the protection of this experiment. Once we merge all buttons will have the new attribute.

Reply:
This allows `core/button` blocks to use an actual `<button>` element - something that should have been implemented from the beginning for button blocks. The fact that we have a "button" block which adds a link instead of a button is less than ideal, and users should at least be able to add an actual button if they need it.
No option is exposed to the UI, and IMO, even if the forms blocks get reverted, this change should remain.

That being said, I do understand your point... and you're right. This change may be a prerequisite for this PR, but it doesn't strictly belong in it. I pushed a separate PR in https://github.com/WordPress/gutenberg/pull/54206, with just this change.

Alternatively, we _could_ use an `<a>` element and then handle things with JS on the front by adding a script but that would be a bit hacky and I'd like to avoid it if possible 😅

Reply:
#54206 was just merged, so the button block changes are no longer included in this PR 🎉